oozie-to-airflow/mappers/base_mapper.py

```diff
@@ -63,19 +57,30 @@ def required_imports() -> Set[str]:
         """
         raise NotImplementedError("Not Implemented")
 
-    def get_task_id(self) -> str:
+    @property
+    def first_task_id(self) -> str:
         """
-        Returns the task_id of the operator, this can be handy if any prepare
-        statements need to be used or any reordering of operators.
+        Returns task_id of first task in mapper
         """
-        return self.task_id
+        return self.name
+
+    @property
+    def last_task_id(self) -> str:
+        """
+        Returns task_id of first task in mapper
+        """
+        return self.name
```

Comment:
I assume this is just a rough sketch, because `last_task_id` does the same as `first_task_id`, but do we need such 2 methods in general? Will we have situations where a mapper has multiple tasks? We should probably discuss this.

Reply:
We discussed this comment in real life.

Reply:
Yes, consider this resolved